packages/tests/src/core-utils/date.ts

```diff
@@ -0,0 +1,29 @@
+import { millisecondsToTime, secondsToTime } from '@peertube/peertube-core-utils'
```

Comment:
Can you also add a barrel file `index.ts` in `core-utils` directory?

Reply:
Alright, but what's this barrel file should export ?

Reply:
`export * from './date.js'`

Reply:
Humm... I'm not sure to understand the purpose of these in the tests folder. In some of the tests folder (for example in `external-plugins`) I'm seeing  `imports` instead of `exports` in the barrel file which might make more sense :thinking:

Reply:
Oup's sorry my bad, you're right!

Reply:
Should I go with a barrel file with `imports` or with no barrel file ?

Reply:
Use `imports` for consistency, even if I plan to remove them in the future (I try to progressively remove barrel files from the project)

Reply:
Well If it's a progressive work, we could begin with not adding this barrel file if it's not necessary !